pom.xml

```diff
@@ -55,21 +55,11 @@
 						<artifactId>groovy-eclipse-compiler</artifactId>
 						<version>2.9.1-01</version>
 					</dependency>
-					<!-- for 2.8.0-01 and later you must have an explicit dependency on
-						groovy-eclipse-batch -->
 					<dependency>
 						<groupId>org.codehaus.groovy</groupId>
 						<artifactId>groovy-eclipse-batch</artifactId>
-						<version>2.3.7-01</version>
-						<!-- or choose a different compiler version -->
-						<!-- <version>2.1.8-01</version> -->
-						<!-- <version>1.7.10-06</version> -->
+						<version>1.8.6-01</version>
```

Comment:
This one produces:

The type java.util.Map$Entry cannot be resolved. It is indirectly referenced from required .class files. No idea why.